### DISCUSSION TITLE
I lost an hour of work after an internal error hung the application aproximally at 9:30 p.m. Argentina time

### DISCUSSION THREAD

**Discussion by cesarverdes** [uxbox/uxbox#1645] [General]
I was giving penpot a try, and after a couple of hours of work (a learning) I lost an hour of work after an internal error hung the app at about 9:30pm. Argentine time.
How frequent are these internal errors? After logging in again, strangely the project is in the state of about an hour before the server error. Is there a way to recover lost work?
:(

**Comment by claragvinola** [comment]
Hi cverdes! Can you give us more info? your OS, etc. And if you can also tell us your user, we can figure out what happened and if it's possible to recover your work.
It's not frequent this kind of error, we're so sorry.

**Reply by do-kevin** [reply]
I am also experiencing a similar problem today at 10:30pm PST. I am using Penpot on Brave browser and on Windows 10. Could not export to Penpot file or artboard to PDFs when it was able to successfully load in.

**Comment by BenGraham-Git** [comment]
For what it's worth I just had this issue and I thought the file was unsalvageable but upon selecting "open in new tab" the file works fine. It seems the problem is tied to a single tab. It's an easy workaround but it was irritating till I learned of the solution. Hoping maybe this helps someone else who searches this same error.
